orderly_web/config.py

```diff
@@ -113,6 +113,7 @@ def __init__(self, path, dat):
         self.images = {
             "orderly": config_image_reference(dat, ["orderly", "image"]),
             "web": config_image_reference(dat, ["web", "image"]),
+            "user-cli": config_image_reference(dat, ["user-cli", "image"]),
```

Comment:
The main suggestion I have is that we move this to `admin` rather than `user-cli` - that it is a "cli" is an implementation detail and the way it is currently exposed to the `orderly-web` deploy tool is as `admin`.  It would then probably make sense to move `orderly_web/users.py` to `orderly_web/admin.py`